python-sdk/src/astro/airflow/datasets.py

```diff
@@ -7,3 +9,26 @@
     # Airflow < 2.4
     Dataset = object
     DATASET_SUPPORT = False
+
+
+def kwargs_with_datasets(
+    kwargs: dict | None = None,
+    input_ds: list[Dataset] | Dataset | None = None,
```

Comment:
Have you considered having arguments with more explicit names `input_datasets` and `output_datasets`?

Reply:
I did but it was just too verbose. I can change them though

Reply:
done in https://github.com/astronomer/astro-sdk/pull/808/commits/3e2ab23827dddbe3d2850e631d3d052ca38e7dbb